src/Core/src/Handlers/MenuFlyoutItem/MenuFlyoutItemHandler.iOS.cs

```diff
@@ -12,8 +12,46 @@ public partial class MenuFlyoutItemHandler
 	{
 		static Dictionary<int, IMenuElement> menus = new Dictionary<int, IMenuElement>();
 
+		bool IsInContextFlyout()
+		{
+			IElement? current = VirtualView;
+			while (current != null)
+			{
+				if (current is Microsoft.Maui.IContextFlyout)
+				{
+					return true;
+				}
+				current = current.Parent;
+			}
+			return false;
+		}
+
 		protected override UIMenuElement CreatePlatformElement()
 		{
+
+			// REVIEW: I don't like this code, but couldn't come up with anything better. It feels right to
```

Comment:
I think this was just an oversight and I was unaware that `UIAction` can just be used in the place of `UICommand`

https://github.com/dotnet/maui/issues/9332